domain/src/main/kotlin/no/nav/su/se/bakover/domain/beregning/Fradrag.kt

```diff
@@ -27,7 +28,19 @@ data class Fradrag(
     val id: UUID = UUID.randomUUID(),
     val type: Fradragstype,
     val beløp: Int,
-    val beskrivelse: String? = null
+    val fraUtlandInntekt: FraUtlandInntekt?,
+    val delerAvPeriode: DelerAvPeriode?
 ) {
     fun perMåned(): Int = beløp / 12
 }
+
+data class FraUtlandInntekt(
+    val beløpUtenlandskValuta: Int,
+    val valuta: String,
+    val kurs: Int
+)
+
+data class DelerAvPeriode(
```

Comment:
Jeg har falt litt ut av domenet her ser det ut til. Hvilken periode refereres det til her og hva er konsekvensene?

Reply:
Eller er det bare at en kan ha visse ting som inntekt etc. i deler av en stønadsperiode og skal ha mulighet til å spesifisere det? Mens default er at det gjelder for hele perioden?

Reply:
man kan for eksempel ha fradrag i perioder fra mars til juni. Disse skal da trekkes fra på en eller annen måte. blir kanskje enten over total-summen de får utbetalt, eller fra de spesifikke månedene de får SU. Skal se om jeg finner på et bedre navn